app/src/examples/PendulumExample.tsx

```diff
@@ -166,20 +166,18 @@ export default function SpringExample() {
           </Animated.View>
         </View>
       </GestureDetector>
-      <React.Fragment>
-        {fields.map((item) => {
-          return (
-            <InputField
-              fieldName={item.fieldName}
-              value={item.value}
-              setValue={(value) => {
-                item.setValue(value);
-              }}
-              key={item.fieldName}
-            />
-          );
-        })}
-      </React.Fragment>
+      {fields.map((item) => {
+        return (
+          <InputField
+            fieldName={item.fieldName}
+            value={item.value}
+            setValue={(value) => {
+              item.setValue(value);
+            }}
```

Comment:
does it work this way?
```suggestion
            setValue={item.setValue}
```